new-pipeline/src/dashboards.js

```diff
@@ -1073,16 +1073,17 @@ function buildDictionaryURL(metric, channel, description) {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
-  var descr = metric;
+  var descrheading = metric;
   if (description && (description.length > 0)) {
     descr = description;
   }
 
   var div = $("<div>", {
     class: "text-center",
   });
+  div.append($("<h3>").text(descrheading));
```

Comment:
I think we can just directly use `...text(metric));` here now?

Reply:
Sure. :)